Resources/Prototypes/species.yml

```diff
@@ -30,11 +30,18 @@
   dollPrototype: MobSlimePersonDummy
   skinColoration: Hues
 
+- type: species
+  id: Skeleton
+  name: Skeleton
+  roundStart: true #touch this and feel my wrath | bet --moony | there will be violence --emo
```

Comment:
I don't feel these are RP friendly enough to be a roundstart and if any maintainers disagree I'm gonna ask why.

Reply:
Yes i'm aware and i've seen the err in my ways.

Reply:
i had a moment and i'm changing it back

Reply:
a